akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -676,10 +676,14 @@ private[akka] class ShardRegion(
 
   def register(): Unit = {
     coordinatorSelection.foreach(_ ! registrationMessage)
-    if (shardBuffers.nonEmpty && retryCount >= 5)
-      log.warning(
+    if (shardBuffers.nonEmpty && retryCount >= 5) coordinatorSelection match {
+      case Some(actorSelection) ⇒ log.warning(
         "Trying to register to coordinator at [{}], but no acknowledgement. Total [{}] buffered messages.",
-        coordinatorSelection, shardBuffers.totalSize)
+        actorSelection, shardBuffers.totalSize)
+      case None ⇒ log.warning(
+        "No coordinator found to register. Probably, no seed-nodes configured and manual cluster join not performed? Total [{}] buffered messages.",
```

Comment:
Below cases are possible for `coordinatorSelection` being `None` ?

1. seed-node not configured and manual configuration is not performed
2. members are joining but not yet `Up`
3. general network problem

I think the original issue #24295 intended to make 1. more obvious, and distinguished from the case where `coordinatorSelection` is `Some()`. However, I'm not sure how to put these 1 ~ 3 cases into a single warning message with reasonable length... so, what I think could do are:

- put a lengthy message saying 1, 2, and 3 are possible
- or, link to a certain section of the doc explaining 1, 2 and 3

Reply:
I think this distinction between None and Some makes it much more clear what the reason might be. Not Up yet should be rare, since coordinator is on the oldest.

One additional hint to the log message of the Some case could be to include something if it's unreachable:
```
val coordinatorIsUnreachable = cluster.state.unreachable(membersByAge.head)
```